### PR TITLE
Lien shark hw2 attempt1

### PR DESCRIPTION
**Pull request by LienShark**
---
A good PR should include the following parts:

- [ ] Provide a clear title
- [ ] Provide meaningful descriptions in the PR and the commit logs
- [ ] Name your branch like ``<github-user-name>-hw#``, e.g., ``myname-hw2-attempt1``
- [ ] Merge from your specific branch (***not master***) to the `hw*` branch (***not master***, again)
- [ ] Exclude irrelevant files, such as binaries, text files, or dot files
- [ ] Make sure tests/CI pass

### CODE REVIEW

File: project/LienShark/Readme.rst [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                  if it can run successfully under extremely large matrices.
+* Week 5 (11/18): Integrate API and complete documentation.
+* Week 6 (11/25): Try and develop mods for different applications like Finance and NLP
+* Week 7 (12/02): Correct potential problems and optimize algorithm performance.
+* Week 8 (12/09): Write reports and prepare final presentations.
+
+References
+==========
+1.Matrix Theory - basic results and techiques 2nd edition by Fuzhen Zhang
+
```
Comment (ThreeMonth03) [inline project/LienShark/Readme.rst]: Please don't submit your project proposal to this pull request.

**Review by ThreeMonth03 — CHANGES_REQUESTED** [review]
- [ ] Remove the unrelated files in hw2.

**Comment by ThreeMonth03** [comment]
By the way, you could try to rebase and squash the commit to make your git history clear.

File: project/LienShark/Readme.rst (hunk 1 above)
Comment (LienShark) [reply project/LienShark/Readme.rst]: Do you mean the  unrelated files you refer to as the readme file from project/LienShark/readme.rst?

Comment (ThreeMonth03) [reply project/LienShark/Readme.rst]: > Do you mean the unrelated files you refer to as the readme file from project/LienShark/readme.rst?

Yes, please remove your project proposal in this pull request.

**Comment by LienShark** [comment]
@ThreeMonth03 I remove the unrelated files, please you check it , thanks TA

**Comment by ThreeMonth03** [comment]
> @ThreeMonth03 I remove the unrelated files, please you check it , thanks TA

Could you please submit the url of this pull request to E3 entry?

**Comment by LienShark** [comment]
@ThreeMonth03 of course! I will do this right now, thanks

**Review by ThreeMonth03 — APPROVED** [review]
LGTM.

**Pull request merged**